cmd/analyze/main.go

```diff
@@ -99,24 +99,10 @@ func main() {
 
 	sb := sandbox.New(manager.Image(), sbOpts...)
 	defer sb.Clean()
-	results := make(map[string]*dynamicanalysis.Result)
-	for _, phase := range manager.DynamicPhases() {
-		result, err := dynamicanalysis.Run(sb, pkg.Command(phase))
-		if err != nil {
-			log.Fatal("Analysis phase failed",
-				log.Label("phase", phase),
-				"error", err)
-		}
-		results[phase] = result
-		// Abort if install failed. No need to continue.
-		if result.Status != dynamicanalysis.StatusCompleted {
-			log.Error("Analysis phase failed. Cannot continue.",
-				log.Label("phase", phase),
-				log.Label("ecosystem", *ecosystem),
-				log.Label("name", *pkgName),
-				log.Label("version", *version))
-			break
-		}
+
+	results, err := analysis.RunDynamicAnalysis(sb, pkg, true)
+	if err != nil {
+		log.Fatal("Aborting due to run error")
```

Comment:
I know it's logged internally, but err should be included in the logging here as well.

Reply:
Sure